release_notes/43-release-notes.adoc

```diff
@@ -271,6 +272,27 @@ Vulnerability deferral management for host (`/node`) and platform (`/cluster`) v
 
 * This release contains updates to the versions of golang and go-git used in {product-title-short}.
 
+[id="resolved-in-version-434_{context}"]
+=== Resolved in version 4.3.4
+
+*Release date*: 19 January 2024
+
+This release contains the following bug fixes and enhancements.
+
+The following fixes and enhancements for integration with Jira and Jira Cloud are implemented:
+
+* The priority is now correctly set on Jira issues created by {product-title-short}.
+* The {product-title-short} integration with Jira Cloud successfully completes.
```

Comment:
This sentence is a little confusing because it's not clear what is completing.  Can we reference the creation of the integration somehow?  

The ACS integration with Jira Cloud ...:

* successfully completes creation
* successfully creates
* can successfully be created.

Reply:
Thanks, I agree that it's awkward - rewrote!